projects/optic/src/commands/oas/operations/diffs/visitors/request-body.test.ts

```diff
@@ -147,9 +147,9 @@ describe('visitRequestBody', () => {
     ]).toHaveLength(0);
     expect([
       ...visitRequestBody(subtypeMisMatch, requestBodySpec),
-    ]).toHaveLength(1);
+    ]).toHaveLength(0);
```

Comment:
Before this would allow you to still learn a body if the content type didn't match. Not sure we actually want this so I changed the test. It just causes weird things to be learned later on.